javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,243 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj){
+  // Solution code here...
+  Object.entries(obj).forEach(entry => {
+    console.log(entry);
+    console.log(entry[0]);
+    let key = entry[0];
+    let value = entry[1];
+    const li = document.createElement('li');
+    li.textContent = `${key}: ${value}`;
+    return li;
```

Comment:
You are on the right track here 👍 

Perhaps we can simplify this a bit by using `map()` instead of `forEach()`. 

- On line 23 you can use a  `return` since we know we need to return an array of key value pairs (forEach won't return anything of course, but map does give us an arr).
- After we can create our key value pairs as html list items